### DISCUSSION TITLE
Icon request: Capslock

### DISCUSSION THREAD

**Discussion by ghost** [FortAwesome/Font-Awesome#21198] [Icon Requests]
I would like to see the Capslock icon :-)

![images](https://cloud.githubusercontent.com/assets/5934247/2870734/91be5570-d2dc-11e3-8fbd-03a5fcafd1a6.jpeg)

**Comment by kryshnar** [comment]
Or like it : 
![](http://i55.tinypic.com/wvz6af.png)

**Comment by leogouveia** [comment]
Since 2014 waiting for a capslock icon

**Comment by deyanpeychev00** [comment]
A Google search led me to this page 😆 

@robmadole please consider this, it's a very long-waited feature reuqest.

**Comment by torrobinson** [comment]
Likewise, I think this would be nice to have. Enough people are puttings caps-lock warnings in login/auth forms that this would be helpful to have!

In the meantime, `square-up` isn't that bad:
![image](https://user-images.githubusercontent.com/4401999/120813558-3a65e300-c50b-11eb-8d06-13a1362243e7.png)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
